src/nectarchain/calibration/container/waveforms.py

```diff
@@ -136,138 +157,196 @@ def check_events(self):
         has_changed = True
         previous_trigger = None
         n_events = 0
-        for i,event in enumerate(self.__reader):
-            if previous_trigger is not None :
-                has_changed = (previous_trigger.value != event.trigger.event_type.value)
+        for i, event in enumerate(self.__reader):
+            if previous_trigger is not None:
+                has_changed = previous_trigger.value != event.trigger.event_type.value
             previous_trigger = event.trigger.event_type
-            if has_changed : 
-                log.info(f"event {i} is {previous_trigger} : {previous_trigger.value} trigger type")
-            n_events+=1
+            if has_changed:
+                log.info(
+                    f"event {i} is {previous_trigger} : {previous_trigger.value} "
+                    f"trigger type"
+                )
+            n_events += 1
         return n_events
 
-
-
-    def load_wfs(self,compute_trigger_patern = False):
-        """mathod to extract waveforms data from the EventSource 
+    def load_wfs(self, start, end, compute_trigger_patern=False):
+        """mathod to extract waveforms data from the EventSource
 
         Args:
-            compute_trigger_patern (bool, optional): To recompute on our side the trigger patern. Defaults to False.
+            compute_trigger_patern (bool, optional): To recompute on
+            our side the trigger patern. Defaults to False.
         """
-        if not(hasattr(self, "wfs_hg")) : 
+        if not (hasattr(self, "wfs_hg")):
             self.__init_arrays()
 
-        wfs_hg_tmp=np.zeros((self.npixels,self.nsamples),dtype = np.uint16)
-        wfs_lg_tmp=np.zeros((self.npixels,self.nsamples),dtype = np.uint16)
-
-        n_traited_events = 0
-        for i,event in enumerate(self.__reader):
-            if i%100 == 0:
-                log.info(f"reading event number {i}")
-
-            self.event_id[i] = np.uint16(event.index.event_id)
-            self.ucts_timestamp[i]=event.nectarcam.tel[WaveformsContainer.TEL_ID].evt.ucts_timestamp
-            self.event_type[i]=event.trigger.event_type.value
-            self.ucts_busy_counter[i] = event.nectarcam.tel[WaveformsContainer.TEL_ID].evt.ucts_busy_counter
-            self.ucts_event_counter[i] = event.nectarcam.tel[WaveformsContainer.TEL_ID].evt.ucts_event_counter
+        wfs_hg_tmp = np.zeros((self.npixels, self.nsamples), dtype=np.uint16)
+        wfs_lg_tmp = np.zeros((self.npixels, self.nsamples), dtype=np.uint16)
 
+        # n_traited_events = 0
+        event_start, event_end = start, end
+        print(event_start, event_end)
```

Comment:
Plain prints should be avoided, to the benefit of the logging system.

Reply:
Agreed! Its remnant from the notebook 😀!